pages/implementations/earl-reports.md

```diff
@@ -4,14 +4,14 @@ title: Reporting Format
 
 ## Contribute An Implementation
 
-If you developed an accessibility tool or a testing methodology, and would like to have your implementation included in the ACT-R website, there are two ways you can do so.
+If you developed an accessibility tool or a testing methodology and would like to have your implementation included in the WAI website, there are two ways you can do so.
 
-1. If you have a tool that can return a data format, you will need to run your tests against the [ACT-R test cases](../testcases/) and [submit a report](../reporting/).
-2. If you use manual test methodology, where you fill results into some report template or tool, you can [Use the WCAG-EM Report Tool](../wcag-em-tool/) instead to produce implementation reports.
+1. If you have a tool that can return a data format, you will need to run your tests against the [test cases](../testcases/) and [submit a report](../reporting/).
```

Comment:
Should "test cases" point to `https://www.w3.org/WAI/content-assets/wcag-act-rules/testcases.json` instead? (just as in [the other change of this PR](https://github.com/act-rules/act-rules.github.io/pull/1837/files#diff-fda9e9325d36a6229d4376045dfb55529cff6a6117fae51b9d4d89a16cfb85c3R7))?

Reply:
No, this is linking to the testcases page, that page describes the format. The testcases page itself does link to the new testcases.json file.

Reply:
Ah yes 🤦